doc/user/content/get-started/quickstart.md

```diff
@@ -12,299 +12,703 @@ aliases:
+      FROM seller_credits
+
+      UNION
 
-    If you `SELECT` from this view, you'll get many results, and results that
-    are changing as new data is generated. This makes it hard to eyeball
-    whether user funds _really_ add up, in the first place.
+      SELECT 0 AS credits, SUM(debits) AS debits
+      FROM buyer_debits
+    );
+
+    ```
 
-1. To double check, you can write a diagnostic query that makes it easier to
-spot that results are correct and consistent. As an example, the total credit
-and total debit amounts should always add up.
+    To verify that the total credits equal the total debits, select from
+    `funds_movement`:
 
     ```mzsql
-    SELECT SUM(credits), SUM(debits) FROM funds_movement;
+    SELECT *
+    FROM funds_movement
+    ;
     ```
 
-    You can also `SUBSCRIBE` to this query, and watch the sums change in lock step as auctions close.
+    To see that the sums always equal even as new data comes in, you can `SUBSCRIBE` to this query:
 
     ```mzsql
     SUBSCRIBE TO (
-        SELECT SUM(credits), SUM(debits) FROM funds_movement
+      SELECT *
+      FROM funds_movement
+    );
+    ```
+
+    - As new data comes in and auctions complete, the `total_credits` and
+     `total_debits` values should change but the `total_difference` should
+     remain `0`.
+
+    - To cancel out of the `SUBSCRIBE`, click **Stop streaming**.
+
+    **Optional exercise*. Alternatively, you can use a common table expression
+    (CTE) to implement the view.
+
+    ```mzsql
+    CREATE VIEW funds_movement_cte_view AS
+    SELECT * FROM (WITH cr AS
+      (
+        SELECT SUM(credits) AS credits, 0 AS debits FROM seller_credits
+        UNION
+        SELECT 0, SUM(debits) FROM buyer_debits
+      ),
+      tot AS
+      (
+        SELECT SUM(credits) AS total_credits, SUM(debits) AS total_debits FROM cr
+      )
+      SELECT total_credits, total_debits, total_credits - total_debits AS
+      total_difference
+      FROM tot
     );
+
+    SUBSCRIBE TO (SELECT * FROM funds_movement_cte_view);
     ```
 
-   It is never wrong, is it?
 
-## Step 5. Clean up
+## Step 7. Additional exercises
+
+1. The views in the quickstart used the `wins_by_item` index on the
+   `winning_bids` view.
+
+   Depending upon your query usage patterns, creating an index only on the
+   view(s) from which you will serve results may be preferred. For example,
+   assume you only serve queries from auction flippers and all the other views
+   serve only as building blocks to the `flippers` views.
+
+   In that case, you may want to
+
+   1. [`DROP`](/sql/drop-index/) the `wins_by_item` index:
+
+      ```mzsql
+      DROP index wins_by_item;
+      ```
+
+   1. Create an index on `flippers`.
+
+      ```mzsql
+      CREATE index flippers_flipper_id ON flippers(flipper_id);
+      ```
+
+   1. Run a query on `flippers`.
+
+      ```mzsql
+      SELECT *
+      FROM flippers
+      WHERE flipper_id = 450;
+      ```
+
```

Comment:
Thinking of adding that it might be that index on the `winning_bids` view might still be better, but that it all depends.